src/openms/include/OpenMS/KERNEL/MSSpectrum.h

```diff
@@ -70,6 +68,7 @@ namespace OpenMS
     public RangeManager<1>,
     public SpectrumSettings
   {
+     friend class MSSpectrumHelper;
```

Comment:
Yep, so the friend relation is needed because the helper needs access to the DataArrays of a spectrum, which are protected, to clone the peaks from a PeakSpectrum into a `MSSpectrum<RichPeak1D>`